Cargo.toml

```diff
@@ -214,7 +214,10 @@ feat_os_windows_legacy = [
 [workspace]
 
 [dependencies]
-uucore   = "0.0.2"
+lazy_static = { version="1.3" }
+textwrap = { version="=0.11.0", features=["term_size"] } # !maint: [2020-05-10; rivy] unstable crate using undocumented features; pinned currently, will review
```

Comment:
Shouldn't we just use `term_size` directly?

Reply:
Well, I'm using this for the text wrap code, not the terminal size.